bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -177,6 +176,11 @@ where
 		}))
 	});
 
+	io.add_sync_method("bitacross_getScheduledEnclaveList", move |_: Params| {
+		debug!("worker_api_direct rpc was called: bitacross_getScheduledEnclaveList");
+		Ok(json!(format!("{:?}", &GLOBAL_SCHEDULED_ENCLAVE.registry.read())))
```

Comment:
You should embed it with `RpcReturnValue` to be consistent with most of methods.